specification/components/schemas/codings/ServiceRequest-Cancellation-Reason.yaml

```diff
@@ -0,0 +1,45 @@
+type: object
+description: e-RS representation of the reason for the state of the ServiceRequest.
+required:
+  - url
+  - valueCoding
+properties:
+  url:
+    type: string
+    enum:
+      - 'reason'
+    example: 'reason'
+  valueCoding:
+    type: object
+    required:
+      - system
+      - code
+    properties:
+      system: 
+        type: string
+        enum:
+          - 'https://fhir.nhs.uk/CodeSystem/eRS-ReferralCancellationReason'
+        example: 'https://fhir.nhs.uk/CodeSystem/eRS-ReferralCancellationReason'
```

Comment:
Just thinking about naming.

Are the e-RS Appointment Request Cancellation Reasons used for A&G too?

Just thinking in the FHIR world this is a ServiceRequest. If the e-RS reasons are used for A&G too then maybe this is just `RequestCancellationReason`?